### ISSUE TITLE
Outdated Cloudflare DNS Record Args

### ISSUE THREAD

**Issue by zdenham** [sst/ion#876]
Error message is here: https://github.com/sst/ion/issues/872

Cloudflare deprecated "value" in favor of "content" in the record object.

See: https://github.com/cloudflare/terraform-provider-cloudflare/issues/3590

The fix for me was

```
sst.cloudflare.dns({
    transform: {
      record: (args) => {
        // @ts-ignore - types are not up to date
        args.content = args.content || args.value;
        args.value = undefined;
      },
    },
  })
 ```

**Comment by benjamindell** [comment]
I'm also seeing this issue. Currently unable to deploy as a result.

**Comment by jayair** [comment]
@benjamindell does the `ts-ignore` workaround work for you?

**Comment by bjorntechTobbe** [comment]
I am also seeing this issue. Using the ts-ignore when calling cloudflate.dns is working but in the case where the value/content needs the url from the frontend as input to cloudflare.dns.createRecord (or createAlias) there is no way to change from value to content. When reading the issue/PR in the pulumi git it looks like this was a breaking change introduced by mistake and would have been fixed.

**Comment by zdenham** [comment]
@bjorntechTobbe another option to fix is pin the cloudflare version e.g. 

https://github.com/sst/ion/issues/872#issuecomment-2294045291

**Comment by bjorntechTobbe** [comment]
Wow,thanks @zdenham, that worked!

**Comment by fwang** [comment]
Fixed in v3.0.80. Updated the default Cloudflare provider version.
